package.json

```diff
@@ -47,6 +47,9 @@
   "pnpm": {
     "overrides": {
       "ipfs-http-client": "^60"
+    },
+    "patchedDependencies": {
+      "graphql-codegen-plugin-typescript-swr@0.8.5": "patches/graphql-codegen-plugin-typescript-swr@0.8.5.patch"
```

Comment:
patch a dep issue and opened two PRs https://github.com/croutonn/graphql-codegen-plugin-typescript-swr/pulls